packages/slice-machine/src/theme.ts

```diff
@@ -324,8 +324,7 @@ const AppTheme = (): Theme =>
         lineHeight: "16px",
         pl: 2,
         pr: 2,
-        pb: "6px",
```

Comment:
Is this the only use of the secondarySmall button class?

Reply:
Yep. 2 references and they're both update screenshot buttons

Reply:
Okay, Is that the only change required by the ticket?

Reply:
Yeah, the button height was the only comment on the ticket from the back from validation